### PR TITLE
About api

### PR DESCRIPTION
**Pull request by joshmoore**
New page intended to live at http://idr-demo.openmicroscopy.org/about/api.html which will initially display a version of https://github.com/IDR/idr-notebooks/blob/master/notebooks/IDR_API_example_script.ipynb

Eventually, this notebook should be checked out automatically and rendered and other info on how to get started using the IDR API should be included.

### CODE REVIEW

**Comment by joshmoore** [comment]
This likely requires:

```
+
+  - name: Install Kramdown
+    gem:
+      name: kramdown
+      state: present
+      user_install: no
+      include_doc: no
+    become: yes
```

File: _includes/IDR_API_example_script.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+
+## Example of using the IDR web API
+
+OMERO.web uses a default session backend authentication scheme for authentication.
+First create a HTTP session using the `requests` library:
```
Comment (aleksandra-tarkowska) [inline _includes/IDR_API_example_script.md]: link `requests` to http://docs.python-requests.org/en/master/user/advanced/#session-objects

File: api.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            <div class="row homepage text-center">
+
+                <div class="medium-12 columns">
+
+                    <h1 class="hero-main-header">IDR API</h1>
+                    <p class="hero-subheader small-10 medium-10 large-10 small-offset-1 medium-offset-1 large-offset-1">
+                    The Image Data Resource (IDR) provides a JSON-based API for accessing all datasets,
+                    thumbnails and metadata. This API is a combination of the standard
+                    <a href="https://www.openmicroscopy.org/site/support/omero/developers/Web.html">OMERO.web API</a>
+                    and newly developed plugins like <a href="https://github.com/ome/omero-mapr/doc">OMERO.mapr</a>.
```
Comment (aleksandra-tarkowska) [inline api.html]: https://github.com/ome/omero-mapr/tree/master/docs

**Comment by joshmoore** [comment]
Merging for testing on clone. Happy to open follow-ons as needed.

**Pull request merged**